pc-app/src/js/interface.js

```diff
@@ -0,0 +1,15 @@
+window.onload = function() {
+    var iframe = document.getElementById('emulated-website');
```

Comment:
These ids don't match the HTML. I think you wrote resizing with CSS already, so these can probably be removed.

Reply:
I copied this code from somewhere because I wanted to have something like a mock iframe. This part is not yet implemented, so this code is just for mock test reasons. I will refactor everything related to interface as the next milestone.